_monitoring-plugins/pa/index.md

```diff
@@ -1,73 +1,110 @@
 
-## Get started with PerfTop
+To start the performance analyzer RCA agent on a tarball install, run the following command.
+      
+````bash
+OPENSEARCH_HOME=~/opensearch-2.2.1 OPENSEARCH_JAVA_HOME=~/opensearch-2.2.1/jdk OPENSEARCH_PATH_CONF=~/opensearch-2.2.1/bin ./performance-analyzer-agent-cli
+````
 
-The basic syntax is:
+The below command is used to enable the performance analyzer plugin and performance analyzer RCA agent.
 
-```bash
-./opensearch-perf-top-<operating_system> --dashboard <dashboard>.json --endpoint <endpoint>
-```
+````bash
+curl -XPOST localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'
+````
 
-If you're using npm, the syntax is similar:
+To shutdown the performance analyzer RCA agent, run the following command.
 
-```bash
-opensearch-perf-top --dashboard <dashboard> --endpoint <endpoint>
-```
+````bash
+kill $(ps aux | grep -i 'PerformanceAnalyzerApp' | grep -v grep | awk '{print $2}')
+````
 
-If you're running PerfTop from a node (i.e. locally), specify port 9600:
+To disable the performance analyzer plugin, run the following command.
 
-```bash
-./opensearch-perf-top-linux --dashboard dashboards/<dashboard>.json --endpoint localhost:9600
-```
+````bash
+curl -XPOST localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": false}'
+````
 
-Otherwise, just specify the OpenSearch endpoint:
+To uninstall the performance analyzer plugin, run the following command.
 
-```bash
-./opensearch-perf-top-macos --dashboard dashboards/<dashboard>.json --endpoint my-cluster.my-domain.com
-```
+````bash
+bin/opensearch-plugin remove opensearch-performance-analyzer
+````
 
-PerfTop has four pre-built dashboards in the `dashboards` directory, but you can also [create your own]({{site.url}}{{site.baseurl}}/monitoring-plugins/pa/dashboards/).
+### Configure performance analyzer
 
-You can also load the pre-built dashboards (ClusterOverview, ClusterNetworkMemoryAnalysis, ClusterThreadAnalysis, or NodeAnalysis) without the JSON files, such as `--dashboard ClusterThreadAnalysis`.
+To configure performance analyzer plugin you will need to edit the `performance-analyzer.properties` configuration file located in the `config/opensearch-performance-analyzer/` directory. Make sure to uncomment the line `#webservice-bind-host` and set it to `0.0.0.0`. You can reference the following example configuration below.
```

Comment:
Add "the" after "configure"
Deleted "located" just say "...file in the..."
Delete "below"